reflex/reflex.py

```diff
@@ -234,50 +187,76 @@ def export(
     # Set the log level.
     console.set_log_level(loglevel)
 
+    # Check which parts of the app to run.
+    export_frontend = not backend_only
+    export_backend = not frontend_only
+
     # Check that the app is initialized.
-    prerequisites.check_initialized(frontend=frontend)
+    prerequisites.check_initialized(frontend=export_frontend)
 
     # Compile the app in production mode and export it.
     console.rule("[bold]Compiling production app and preparing for export.")
 
-    if frontend:
+    if export_frontend:
         # Ensure module can be imported and app.compile() is called.
         prerequisites.get_app()
         # Set up .web directory and install frontend dependencies.
         build.setup_frontend(Path.cwd())
 
     # Export the app.
-    config = get_config()
     build.export(
-        backend=backend,
-        frontend=frontend,
-        zip=zipping,
+        backend=export_backend,
+        frontend=export_frontend,
         deploy_url=config.deploy_url,
     )
 
     # Post a telemetry event.
     telemetry.send("export", config.telemetry_enabled)
+    console.success(
+        """App exported to[bold]backend.zip[/bold] and [bold]frontend.zip[bold]."""
```

Comment:
i think we should keep track of output files and not say we exported a particular file if we didn't (#1285)

```python
outputs = set()
if export_backend:
    if zipping:
        outputs.add("backend.zip")
    else:
        outputs.add("app")
if export_frontend:
    if zipping:
        outputs.add("frontend.zip")
    else:
        outputs.add(".web/_static")
```
...
```python
console.success(
    "App exported to " + " and ".join(f"[bold]{filename}[/bold]" for filename in outputs)
)
```

Reply:
Good call, will change this